doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -18,26 +18,45 @@
-
-Figure 4-1 below shows the architectural components that are described in the subsequent sections of this chapter.
-
-<p align="center"><img src="../figures/ch04_k8s_architecture.png" alt="Kubernetes Reference Architecture" Title="Kubernetes Reference Architecture" width="65%"/></p>
+This chapter describes in detail the Kubernetes Reference Architecture in terms
+of the functional capabilities and how they relate to the Reference Model
+requirements, i.e. how the infrastructure profiles are determined, documented
+and delivered.
+
+The specifications defined in this chapter will be detailed with unique
+identifiers, which will follow the pattern: `ra2.<section>.<index>`, e.g.
+`ra2.ch.001` for the first requirement in the Container Host section.  These
+specifications will then be used as requirements input for the Kubernetes
+Reference Implementation and any Vendor or Community Implementations.
+
+Figure 4-1 below shows the architectural components that are described in the
+subsequent sections of this chapter.
+
+<p align="center"><img src="../figures/ch04_k8s_architecture.png"
+alt="Kubernetes Reference Architecture" Title="Kubernetes Reference
+Architecture" width="65%"/></p>
 <p align="center"><b>Figure 4-1:</b> Kubernetes Reference Architecture</p>
 
 <a name="4.2"></a>
 ## 4.2 Container Host
 
-This section describes the configuration that will be applied to the physical or virtual machine and an installed Operating System.
-In order for a Container Host to be conformant with the Reference Architecture it must be implemented as per the following specifications:
+This section describes the configuration that will be applied to the physical or
+virtual machine and an installed Operating System. In order for a Container Host
+to be conformant with the Reference Architecture it must be implemented as per
+the following specifications:
 
 |Ref|Specification|Details|Requirement Trace|
 |---|---|---|---|
-|`ra2.ch.001`||||
-|`ra2.ch.002`||||
-|`ra2.ch.003`||||
+|`ra2.ch.001`|Huge Pages|Huge Pages must be enabled within the Container Host OS, with at least one each of 2048KiB and 1048576KiB page sizes, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|[infra.com.cfg.004](./chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ch.002`|SR-IOV capable NICs|The physical machine on which the Container Host runs must be equipped with NICs that are SR-IOV capable (e.g. [this list from Intel](../../../../../../../../intel/sriov-network-device-plugin#supported-sr-iov-nics)).|[e.cap.013](./chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
```

Comment:
might want to include a Mellanox link too for proper vendor representation :)

Reply:
I agree!  Ideally there would be a vendor-agnostic list somewhere.  The Intel link above includes Mellanox cards, but perhaps not all of them.

Reply:
yeah my bad, haven't cehcked the link. but for example ConnectX6 series is missing from the list, yep :)
I guess any such list is destined to go out of date so might be better to just state he generic requirement

Reply:
Removed link to Intel doc in https://github.com/cntt-n/CNTT/commit/b90ac99e4a8e53cb1ab94949f32ab66b5578f75d.